### DISCUSSION TITLE
Patching after other package plugins have ran?

### DISCUSSION THREAD

**Discussion by odeland** [cweagans/composer-patches#516] [Q&A]
I am trying to patch a package (statamic/cms) that uses a plugin "pixelfear/composer-dist-plugin" which is used to download extra files (compiled JS in this case). Unfortunately, the patch tries to run before the plugin does its job, so the "dist" files are not there (which I need to patch). I'm wondering if it's possible to choose when the composer-patch will run in the composer cycle. I'm not sure this is even possible at all, which is why I'm asking here.

**Comment by cweagans** [comment]
Unfortunately, this isn't something that can be configuration driven on my end.

I think you'd need to change the [event weights](https://github.com/pixelfear/composer-dist-plugin/blob/master/src/Plugin.php#L25-L31) in https://github.com/pixelfear/composer-dist-plugin to be lower to make this work properly. See also: https://getcomposer.org/doc/articles/plugins.md#event-handler
